noah/src/main/resources/crawler/promedmail_spider.py

```diff
@@ -71,29 +69,34 @@ def start_requests(self):
 		self.s.add(self.archNo)
 		if not os.path.exists(self.directoryName):
 			os.makedirs(self.directoryName)
-		self.completePath = os.path.abspath(self.directoryName)	
-		
-		url =  self.part_url + self.archNo
-		yield SplashRequest(url, self.parse, args={'wait': 0.5})
+			self.completePath = os.path.abspath(self.directoryName)	
+			url =  self.part_url + self.archNo
+			yield SplashRequest(url, self.parse, args={'wait': 0.7})
+		else:
+			self.completePath = os.path.abspath(self.directoryName)	
+			url =  self.part_url + self.archNo
+			yield SplashRequest(url, self.parse, args={'wait': 0.7})
 			
 	def parse(self, response):
 		html=response.body
 		
 		#save the response in an html file
 		filename = response.url.split("/")[-1] + '.html'
 		completeName = os.path.join(self.completePath, filename)
-		with open(completeName, 'wb') as f:
-			f.write(html)
+		if not os.path.isfile(completeName):
+			with open(completeName, 'wb') as f:
+				f.write(html)
+		else:
+			pass
 			
 		#regex to match the archive numbers
 		listId = re.findall(r"\D(\d{8})"+"."+"(\d{5,8})\D", html)
-		print listId
 		for id in listId:
 			concatId = id[0]+"."+id[1]
 			if concatId not in self.s:
 				url = self.part_url + concatId
 				self.s.add(concatId)
-				yield SplashRequest(url, self.parse, args={'wait': 0.5})
+				yield SplashRequest(url, self.parse, args={'wait': 0.7})
```

Comment:
It was extremely fast with 0.5. I was debugging and trying to test it. It can be anything. Its just wait time before sending another request. I does not make any changes.

Done.